wandb/sdk/interface/artifacts.py

```diff
@@ -58,8 +61,6 @@ def bytes_to_hex(bytestr):
 
 
 class ArtifactManifest(object):
```

Comment:
This is sick, I'm not sure there's a good reason for this other than it's the "right way" but we could consider making this an actual [abstract class](https://docs.python.org/3/library/abc.html).